var/classes/definition_BodyStyle.php

```diff
@@ -96,16 +91,14 @@
                  'locked' => false,
                  'style' => '',
                  'permissions' => NULL,
-                 'datatype' => 'data',
-                 'fieldtype' => 'input',
+                 'fieldtype' => '',
```

Comment:
> @dvesh3 @kingjia90 Is ready to review now.

We are waiting for Bernie to check with this https://github.com/pimcore/demo/pull/533#issuecomment-1701187369
TLDR: fieldType should be already removed since 11.0.0, the https://github.com/pimcore/ecommerce-framework-bundle/pull/119 might not even need a deprecation, if we consider that these were just missing the `@internal` tag.
Depending on the outcome, all these line should might need to be cleaned
```suggestion
```

Reply:
@kingjia90 Okay 👍🏼

Reply:
@blankse Had a quick chat with Bernie, and we end up deciding that https://github.com/pimcore/pimcore/pull/15770 should be split, the deprecation/upgrade note should be moved in an new PR for `11.1`.
For https://github.com/pimcore/ecommerce-framework-bundle/pull/119 need to revert the deprecation tags added in `1.0.4` and move them in a PR targeting `1.1`
So for the moment, these `'fieldtype' => '',` would stay, and this PR is ready to be reviewed.

Thank you in advance!

Reply:
@kingjia90 Done
https://github.com/pimcore/pimcore/pull/15770
https://github.com/pimcore/pimcore/pull/15899
https://github.com/pimcore/ecommerce-framework-bundle/pull/124

The PR for 1.1 can be created if this one is merged in 1.0 and 1.x

Reply:
@kingjia90 Created the PR for 1.1 https://github.com/pimcore/ecommerce-framework-bundle/pull/125